src/data/markdown/translated-guides/en/03 Using k6 browser/01 Overview.md

```diff
@@ -13,7 +13,7 @@ This module aims to provide rough compatibility with the Playwright API, so you
 
 <Blockquote mod="note" title="">
 
-To work with the browser module, make sure you are using the latest [k6 version](https://github.com/grafana/k6/releases).
+To work with the browser module, ensure you are using the latest [k6 version](https://github.com/grafana/k6/releases), and install a Chromium-based browser on your machine.
```

Comment:
```suggestion
To work with the browser module, make sure you are using the latest [k6 version](https://github.com/grafana/k6/releases), and install a Chromium-based browser on your machine (such as [Google Chrome](https://www.google.com/chrome/)).
```

nit: I think "make sure" sounds better/more friendly than "ensure

Adding a mention of Google Chrome is just a suggestion, I think it can help if people are not familiar with Chromium (even if they're very similar 😅), but feel free to ignore.

Reply:
👍 Do we have a writing style somewhere? :)

Reply:
@inancgumus I think this would be the relevant page in our style guide: https://grafana.com/docs/writers-toolkit/write/style-guide/voice-tone-guidelines/.

Reply:
That's perfect! Thanks 🙇